examples/03_connectivity/plot_group_level_connectivity.py

```diff
@@ -0,0 +1,222 @@
+# To extract regions time series, we instantiate a
+# :class:`nilearn.input_data.NiftiMapsMasker` object and pass the atlas the
+# file name to it, as well as filtering band-width and detrending option.
+from nilearn import input_data
+
+masker = input_data.NiftiMapsMasker(
+    msdl_data.maps, resampling_target="maps", t_r=2.5, detrend=True,
+    low_pass=.1, high_pass=.01, memory='nilearn_cache', memory_level=1)
+
+###############################################################################
+# Then we compute region signals and extract useful phenotypic informations.
+adhd_subjects = []
+pooled_subjects = []
+site_names = []
+adhd_labels = []  # 1 if ADHD, 0 if control
+for func_file, confound_file, phenotypic in zip(
+        adhd_data.func, adhd_data.confounds, adhd_data.phenotypic):
+    time_series = masker.fit_transform(func_file, confounds=confound_file)
+    pooled_subjects.append(time_series)
+    is_adhd = phenotypic['adhd']
+    if is_adhd:
+        adhd_subjects.append(time_series)
+
+    site_names.append(phenotypic['site'])
+    adhd_labels.append(is_adhd)
+
+print('Data has {0} ADHD subjects.'.format(len(adhd_subjects)))
+
+###############################################################################
+# ROI-to-ROI correlations of ADHD patients
+# ----------------------------------------
+
+###############################################################################
+# The simpler and most commonly used kind of connectivity is correlation. It
+# models the full (marginal) connectivity between pairwise ROIs. We can
+# estimate it using :class:`nilearn.connectome.ConnectivityMeasure`.
+from nilearn.connectome import ConnectivityMeasure
+
+correlation_measure = ConnectivityMeasure(kind='correlation')
+
+###############################################################################
+# From the list of ROIs time-series for ADHD subjects, the
+# `correlation_measure` computes individual correlation matrices.
+correlation_matrices = correlation_measure.fit_transform(adhd_subjects)
+
+# All individual coefficients are stacked in a unique 2D matrix.
+print('Correlations of ADHD patients are stacked in an array of shape {0}'
+      .format(correlation_matrices.shape))
+
+###############################################################################
+# as well as the average correlation across all fitted subjects.
+mean_correlation_matrix = correlation_measure.mean_
+print('Mean correlation has shape {0}.'.format(mean_correlation_matrix.shape))
+
+###############################################################################
+# We display the connectomes of the first 3 ADHD subjects and the mean
+# correlation matrix over all ADHD patients.
+from nilearn import plotting
+
+plot_matrices(correlation_matrices[:4], 'correlation')
+plotting.plot_connectome(mean_correlation_matrix, msdl_coords,
+                         title='mean correlation over 13 ADHD subjects')
```

Comment:
For me title looks misleading. It said 13 subjects but we plot only 4. Could we change it to 4 ?